src/solvers/permute_indices.jl

```diff
@@ -0,0 +1,16 @@
+
+function permute_indices(tn)
```

Comment:
```suggestion
function align_indices(tn)
```
I've started to use the terminology `align` instead of `permute` for ordering the indices of an ITensor/NamedDimsArray in NamedDimsArrays.jl, I think it expresses the intent better.